packages/@stylexjs/babel-plugin/src/shared/stylex-vars-utils.js

```diff
@@ -19,6 +19,8 @@ export type VarsConfig = $ReadOnly<{
 
 const SPLIT_TOKEN = '__$$__';
 
+export const ROOT_VARS_CLASSNAME = 'stylexvars';
```

Comment:
I hardcoded the class name because it felt difficult to understand the snapshots when everything was a hashed string, and this key needs to exist independently from the hashed strings used to identify the product of `defineVars` calls.